nengo/tests/test_node.py

```diff
@@ -139,25 +139,31 @@ def test_circular(Simulator, seed):
     assert np.allclose(sim.data[a_p], sim.data[b_p])
 
 
-def test_function_args_error(Simulator):
+def test_outputparam_errors(Simulator):
     with nengo.Network() as model:
+        # type errors
```

Comment:
This comment is misleading. The next few lines are perfectly fine and don't produce type errors. (Took me a few moments to realize why there is no check for raised exceptions for this lines.)